doc/release/platforms/README.cygwin.rst

```diff
@@ -34,14 +34,13 @@ please let us know at chapel_info@cray.com.
 Required Packages
 -----------------
 
-In order to get the required tools listed in README.prereqs, Cygwin
-users will need to ensure that the following packages are installed in
-addition to the standard set of Cygwin packages:
+In order to get the required tools listed in :ref:`readme-prereqs`,
+Cygwin users will need to ensure that the following packages are
+installed in addition to the standard set of Cygwin packages:
 
     Devel
       + gcc and gcc-g++
       + make
-      + patch
```

Comment:
No, I don't think so. I didn't need to do that on a fresh cygwin install. I think it's an artifact of our machines.

Even if it wasn't, I'd rather wait to add it until somebody pinged us. I doubt we anybody developing on cygwin that will need the test system, and adding the symlink requires admin privileges